transcriptic/cli.py

```diff
@@ -179,6 +179,68 @@ def cli(ctx, api_root, email, token, organization, config):
             pass
 
 
+@cli.command("exec")
+@click.argument("autoprotocol", type=click.File("r"), default=sys.stdin)
+@click.option(
+    "--api", "-a", help="The api endpoint of your scle test workcell instance."
+)
+@click.option(
+    "--workcellId",
+    "-wc",
+    help="The workcell id to use for the device set. This is not permitted along with the `deviceSet` option.",
+)
+@click.option(
+    "--deviceSet",
```

Comment:
nit: Let's keep the option-names consistent by using `snake-case` instead of camelCase